### PR TITLE
Benjamin rogers

### PR DESCRIPTION
**Pull request by thisbenrogers**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -13,6 +14,7 @@ export default class App extends Component {
   }
 
   addToSavedList = movie => {
+    console.log("Saved movie!")
```
Comment (RyanNFreeman) [inline client/src/App.js]: Good idea logging something like this.

File: client/src/App.js [hunk 2]
```diff
@@ -22,7 +24,8 @@ export default class App extends Component {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <Route exact path="/" component={MovieList} />
```
Comment (RyanNFreeman) [inline client/src/App.js]: Nice! Some devs will say `exact=true` but I think it's more common to just write `exact`. Looks good.

File: client/src/Movies/Movie.js [hunk 3]
```diff
@@ -11,7 +12,7 @@ export default class Movie extends Component {
 
   componentDidMount() {
     // change this line to grab the id passed on the URL
-    const id = 1;
+    const id = this.props.match.params.id;
```
Comment (RyanNFreeman) [inline client/src/Movies/Movie.js]: You will see this again in backend so get used to it.

File: client/src/Movies/MovieCard.js [hunk 4]
```diff
@@ -1,7 +1,25 @@
 import React from 'react';
 
 const MovieCard = props => {
-  return;
+  const { title, director, metascore, stars } = props.movie;
```
Comment (RyanNFreeman) [inline client/src/Movies/MovieCard.js]: Love the destructuring. So clean.

Comment (thisbenrogers) [reply client/src/Movies/MovieCard.js]: @RyanNFreeman thanks for all the awesome feedback man, you're a stellar PM

**Pull request merged**